version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.10-SNAPSHOT"
+ThisBuild / version := "0.10.0-SNAPSHOT"
```

Comment:
This is good - we need to use a strict-semver (`major.minor.patch`) version format for the `sbt-release-plugin`, and so we are switching from `0.10` to `0.10.0` - `0.10.0` will correctly be considered as 'newer' than `0.9`